platform/kobo/koreader.sh

```diff
@@ -260,6 +260,7 @@ while [ $RETURN_VALUE -ne 0 ]; do
 
         echo "!!!!" >>crash.log 2>&1
```

Comment:
The CI doesn't seem to have run on your PR? Odd. 

Sidebar, this line triggers https://github.com/koalaman/shellcheck/wiki/SC2129 on newer SC.

Reply:
Looks like it's not just newer SC. Oops. (Why didn't CirlceCI run? >_<)